trillian_admin_api.proto

```diff
@@ -20,7 +20,7 @@ option java_outer_classname = "TrillianApiProto";
 
 package trillian;
 
-import "trillian.proto";
+import "github.com/google/trillian/trillian.proto";
```

Comment:
I've been looking at proto files in other Google APIs as a reference, but none seem to include the repo path in their imports.

See this [search on github.com/googleapis/googleapis](https://github.com/googleapis/googleapis/search?utf8=%E2%9C%93&q=import&type=), for example.

I wonder if we're being a bit unorthodox here. Maybe instead of the full repo path, we could go with just `trillian/` (eg, `trillian/trillian.proto`)?